packages/core/src/types/index.ts

```diff
@@ -63,6 +63,11 @@ export function mapOf(typeName: string) {
   };
 }
 
+export const VendorExtension: NormalizedNodeType = {
+  name: 'VendorExtension',
```

Comment:
Do you think it should be `SpecificationExtension` instead of `VendorExtension`? The name changed in 3.0 and should therefore cover most of the usage?

Reply:
oh, I somehow missed this fact.

Yes, I think `SpecificationExtension` is better. What do you think about `SpecExtension` or just `Extension` though?